projects/plugins/jetpack/CHANGELOG.md

```diff
@@ -4,25 +4,25 @@
 
 ## 11.7-a.3 - 2022-12-12
 ### Enhancements
-- Jetpack: convert core/embed block, videopress variation, to VideoPress video block, on the fly for Simple sites [#27857]
-- Jetpack: remove video block transform to/from videopress/video for Simple sites [#27834]
-- Update Form blocks descriptions [#27819]
-- Update Form block Sidebar to make it simpler [#27820]
+- Form block: update Form blocks descriptions. [#27819]
+- Form block: simplify Form block sidebar to make the UI easier to use. [#27820]
+- Subscriptions block: change the label "email subscribers" to "subscribers" in the pre/post publish panel, as it also includes followers. [#27879]
 - Writing prompts: add filter for whether prompts are enabled or not [#27778]
 
 ### Improved compatibility
-- Jetpack: convert on-the-fly video block instances on dotcom [#27831]
+- VideoPress block: convert on-the-fly video block instances on wordpress.com. [#27831]
+- VideoPress block: convert core/embed block, videopress variation, to VideoPress video block, on the fly. Only relevant for wordpress.com sites. [#27857]
```

Comment:
```suggestion
- VideoPress block: convert core/embed block, videopress variation, to VideoPress video block, on the fly. Only relevant for WordPress.com sites. [#27857]
```

Actually do we put any wpcom specific changelogs to "other"?

Reply:
Not too sure, but I imagine we can since WPcom don't consume these changelogs. I've moved it over (and added in those caps - thanks).